kafka-client/src/main/java/dev/responsive/kafka/api/async/internals/queues/FinalizingQueue.java

```diff
@@ -76,6 +78,8 @@ public AsyncEvent nextFinalizableEvent() {
 
   /**
    * See {@link ReadOnlyFinalizingQueue#waitForFinalizableEvent()}
+   * <p>
+   * Note: blocking API
    */
   @Override
   public AsyncEvent waitForFinalizableEvent() throws InterruptedException {
```

Comment:
nit: name this `waitForNextFinalizableEvent` to keep it consistent with `nextFinalizableEvent`